package.json

```diff
@@ -11,7 +12,8 @@
   "devDependencies": {},
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node server.js",
+    "postinstall": "bower install"
```

Comment:
This `postinstall` script automatically runs after `npm i` so you don't have to do `npm i` and `bower i` to do setup.